packages/packages.go

```diff
@@ -66,26 +70,26 @@ func (d Dir) Packages() map[string]*ast.Package {
 // Import imports path from srcDir, then loads the ast for that package.
 // It ensures that the returned ast is for the package that would be
 // imported by an import clause.
-func (d Dir) Import(path, srcDir string) (string, *ast.Package, error) {
-	pkgInfo, err := build.Import(path, srcDir, 0)
+func (d Dir) Import(path string) (string, *ast.Package, error) {
+	pkgInfo, err := build.Import(path, d.Path(), 0)
```

Comment:
I'm surprised that the `ImportMode` of 0 works since it is not a valid `ImportMode` value (1,2,4,8).
However, it seems that changing this to any even number value and/or `1` makes the test pass. So may be it is irrelevant.
This is the only thing that stumps me